token/core/zkatdlog/crypto/issue/issue.go

```diff
@@ -80,34 +101,60 @@ func (i *IssueAction) GetSerializedOutputs() ([][]byte, error) {
 }
 
 // GetIssuer returns the Issuer of IssueAction
-func (i *IssueAction) GetIssuer() []byte {
+func (i *Action) GetIssuer() []byte {
 	return i.Issuer
 }
 
 // Deserialize un-marshals IssueAction
-func (i *IssueAction) Deserialize(raw []byte) error {
+func (i *Action) Deserialize(raw []byte) error {
 	return json.Unmarshal(raw, i)
 }
 
-// GetCommitments return the Pedersen commitment of (type, value) in the OutputTokens
-func (i *IssueAction) GetCommitments() ([]*math.G1, error) {
-	com := make([]*math.G1, len(i.OutputTokens))
+// GetCommitments return the Pedersen commitment of (type, value) in the Outputs
+func (i *Action) GetCommitments() ([]*math.G1, error) {
+	com := make([]*math.G1, len(i.Outputs))
 	for j := 0; j < len(com); j++ {
-		if i.OutputTokens[j] == nil {
+		if i.Outputs[j] == nil {
 			return nil, errors.New("invalid issue: there is a nil output")
 		}
-		com[j] = i.OutputTokens[j].Data
+		com[j] = i.Outputs[j].Data
 	}
 	return com, nil
 }
 
 // IsGraphHiding returns false, indicating that fabtoken does not hide the transaction graph
```

Comment:
Not relevant to this pr but maybe update the token and say "this implementation of zkat" instead of "fabtoken"

Reply:
okay